src/Collections-Strings/Symbol.class.st

```diff
@@ -210,14 +210,10 @@ Symbol class >> readFrom: strm [
 
 { #category : #cleanup }
 Symbol class >> rebuildSelectorTable [
-	| selectorTable |
-	selectorTable := WeakSet new.
-	CompiledMethod
-		allInstancesDo: [ :method | 
-			| selector |
-			selector := method selector.
-			selector ifNotNil: [ selectorTable add: selector ] ].
-	^selectorTable.
+
+	^ SystemNavigation new allMethods
```

Comment:
#allMethods is quite expensive itself. I wonder is it really faster than #allInstancesDo: in that case?

Reply:
The idea was to avoid allInstancesDo: as it gets slower with the number of objects (not good for large images with lots of data, but not much code).

We are just interested in those symbols that are used as a method name of *installed* methods, not all methods (even though there should be no difference when it is called, I think)